img2mol/inference.py

```diff
@@ -39,14 +46,36 @@ class Img2MolInference(object):
     """
     Inference Class
     """
-    def __init__(self, model_ckpt: Optional[str] = None, device: str = "cpu"):
+    def __init__(
+        self,
+        model_ckpt: Optional[str] = None,
+        device: str = "cuda:0" if torch.cuda.is_available() else "cpu",
+        local_cddd: bool = False
+    ):
         super(Img2MolInference, self).__init__()
+        if local_cddd:
+            self.cddd_inference_model = CDDDInferenceModel()
+        else:
+            self.cddd_inference_model = False
+            self.cddd_server = CDDDRequest()
         self.device = device
         print("Initializing Img2Mol Model with random weights.")
         self.model = Img2MolPlModel()
         if model_ckpt is not None:
             print(f"Loading checkpoint: {model_ckpt}")
             self.model = self.model.load_from_checkpoint(model_ckpt)
+        else:
+            # Model path in package_dir
+            model_path = os.path.join(os.path.split(__file__)[0],
```

Comment:
Storing very large files like the model checkpoints next to the installed python packages is bad practice. For smaller files this would usually be done via package data and `importlib.resources`.

For this package, I would recommend having this as a configurable location, so that the user can decide where it's stored.
To prevent confusion I would default to "not set" and emit an error message that tells the user to provide a model folder.

Reply:
I have changed everything back to the old behaviour where the model path needs to be set manually.

May I ask why you would consider it bad practice? We normally try to manage the model download automatically on the first use and just store it in the package directory. I would have thought that this makes the usage easier for the end-user when they just need to pip install it and are ready-to-go without needing to specify the paths manually.